jackrabbit-spi2dav/src/main/java/org/apache/jackrabbit/spi2dav/RepositoryServiceImpl.java

```diff
@@ -341,15 +353,41 @@ public RepositoryServiceImpl(String uri, IdFactory idFactory,
             throw new RepositoryException(e);
         }
 
+        // TODO: move to ConnectionOptions as well
```

Comment:
I tend to move this option as well!

Reply:
Done in https://github.com/apache/jackrabbit/pull/92/commits/9ef98400bcf513aa3ce1fedabad95a032a3d543e